spec/requests/applicants/omniauth_callbacks_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+RSpec.describe 'applicants omniauth call back', type: :request do
+  around(:example) do |example|
+    OmniAuth.config.test_mode = true
+    example.run
+    OmniAuth.config.mock_auth[:true_layer] = nil
+    OmniAuth.config.test_mode = false
+  end
+
+  let(:token) { SecureRandom.uuid }
+  let(:expires_at) { 1.hour.from_now.round }
+  let(:true_layer_expires_at) { expires_at.to_i }
+  let(:applicant) { create :applicant }
+  let(:identification_tool) { double applicant: applicant }
```

Comment:
💭 If possible avoid the use of doubles with no connection to the real object we're mocking. If we can use `instance_double(TrueLayer::IdentifyApplicant)` instead, as it enforces the methods defined in the double to exist in the original object, which is not the case for a simple `double`.